### ISSUE TITLE
Samplerstate for texture? TextureFiltering for images / textures

### ISSUE THREAD

**Issue by sachssoft** [MyraUI/Myra#470]
Hi, 

The texture of an image button looks too pixelated. I need to set the sampler status to Linear.

There is a private member "SetTextureFiltering" in Myra.Graphics2D.RenderContext.

Any help for me?

**Comment by Newbilius** [comment]
I agree, this is an extremely necessary setting, for example, for the Image component, especially when scale image down.

**Comment by Newbilius** [comment]
Create pull request for this
https://github.com/rds1983/Myra/pull/478

**Comment by dudule73** [comment]
Hello, for sure this would be necessary ! The images are very ugly after resising.
Not implemented in Myra 1.5.9 ? I'm not able to find it...

**Comment by rds1983** [comment]
Now you can set TextureFiltering for all images(MyraEnvironment.ImageTextureFiltering) as well for individual Images(Image.TextureFiltering)
